docs/source/user/aerodyn-olaf/InputFiles.rst

```diff
@@ -312,6 +312,118 @@ of a box of shape 5x20x30 and dimension 1200x300x295.  The grid contains both th
 The two other grids are vertical and horizontal planes containing only the velocity.
 
 
+Advanced Options
+~~~~~~~~~~~~~~~~
+
+
+Advanced options (typically used for developers or beta features) can be placed at the end of the OLAF input file:
+
+
+- These options use the regular format:  `Value Key - Comment`.
+- They can be placed in arbitrary order.
+- They can be commented out with a `!` character at the beginning of the line.
+- Blank lines or unsupported options are ignored (display to screen).
+- If not provided their default value is being used, but the keyword "default" is not available.
+- We recall that these are **beta features** and should mostly be used by developers.
+
+
+The end of the OLAF input file would look as follows:
+
+.. code::
+
+   [...]
+   GridName  GridType  TStart  TEnd     DTGrid    XStart    XEnd   nX    YStart   YEnd    nY    ZStart   ZEnd   nZ
+   (-)         (-)      (s)     (s)      (s)        (m)      (m)    (-)    (m)     (m)     (-)    (m)     (m)    (-)
+   ===============================================================================================
+   --------------------------- ADVANCED OPTIONS --------------------------------------------------
+   ===============================================================================================
+   ! Advanced options may be placed here in arbitrary order, with the regular format:
+   ! Value1   Key1            - Comment1
+   ! Value2   Key2            - Comment2
+   ! Lines starting with `!` are ignored, empty lines are ignored
+   [...] etc
+
+Currently, the advanced options supported as as follows:
+
+.. code::
+
+   ===============================================================================================
+   --------------------------- ADVANCED OPTIONS --------------------------------------------------
+   ===============================================================================================
+   "Panels.vtk"  SrcPnlFile     - Name of VTK file containing source panels {default: ""}
+   1             nSrcPnlUpdate  - How often do src panel updates (in time steps of OLAF), {default: 1}
+   True          Induction      - Compute induction, {default: True} 
+   True          InductionAtCP  - Compute induced velocities at nodes or CP, {default: True} 
+   True          WakeAtTE       - Start the wake at the trailing edge, or at the LL, {default: True} 
+   False         DStallOnWake   - Dynamic stall has influence on wake, {default: False} 
+   0.75          kFrozenNWStart - Fraction of wake induced velocity at start of frozen wake, {default: 0.75} 
+   0.5           kFrozenNWEnd   - Fraction of wake induced velocity at end of frozen wake, {default: 0.5} 
+   0.0           zGround        - Ground height {default: 0.0}
+   0.1           zGroundPush    - Ground push back {default: 0.1}
```

Comment:
I'm not sure how these options are sorted, but they are not alphabetical or sorted by topic.  Suggest sorting in terms of their functionality, e.g., lifting line or source panel or ....

Reply:
they are sorted by functionality, but they are definitely obscure.